### PR TITLE
Change lazy-loading-attr to also cover iframes

### PR DESCRIPTION
**Pull request by zcorpan**
See #840

### CODE REVIEW

**Review by tantek — CHANGES_REQUESTED** [review]
These changes look good.

I would additionally suggest updating the mdnUrl to https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading#images_and_iframes which covers both images and iframes

Ideally we'd link to both mozBugUrls (image and iframe lazy loading) but I'm not sure if multiple bug URLs are supported

File: activities.json [hunk 1]
```diff
@@ -886,16 +886,16 @@
   },
   {
     "ciuName": "loading-lazy-attr",
-    "description": "Enabling images to be fetched at a later time, when the user is likely to look at them.",
+    "description": "Enabling images and iframes to be fetched at a later time, when the user is likely to look at them.",
     "id": "loading-lazy-attr",
     "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading",
```
Comment (zcorpan) [inline activities.json:891]:
```suggestion
    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading#images_and_iframes",
```

**Comment by zcorpan** [comment]
@tantek changed mdnUrl. Yeah I think we can only point to one bug.

**Review by tantek — APPROVED** [review]
LGTM

**Pull request merged**